bindings/python/dnet_client

```diff
@@ -306,11 +306,319 @@ def process_iterator():
+    parser.add_option('--hex', action='store_true', dest='hex', default=False,
+                      help='Specifies that the key used with --key is hex string')
+
+    add_node_options(parser)
+
+    options, _ = parser.parse_args()
+
+    key = elliptics.Id.from_hex(options.key) if options.hex else options.key
+
+    session = create_session(parser, options, new_session=True, single_remote=False)
+
+    session.exceptions_policy = elliptics.exceptions_policy.no_exceptions
+    session.set_filter(elliptics.filters.all_final)
+
+    results = []
+    for group in session.groups:
+        session.groups = [int(group)]
+        results.append(session.lookup(key))
+
+    result = OrderedDict()
+    for r in results:
+        r = r.get()[0]
+        group_result = OrderedDict(status=r.status)
+
+        if r.path is not None:
+            group_result['path'] = r.path
+
+        if r.record_info is not None:
+            group_result['record_info'] = OrderedDict((
+                ('record_flags', elliptics.dump_record_flags(r.record_info.record_flags)),
+                ('user_flags', r.record_info.user_flags),
+                ('json_timestamp', str(r.record_info.json_timestamp)),
+                ('json_offset', r.record_info.json_offset),
+                ('json_size', r.record_info.json_size),
+                ('json_capacity', r.record_info.json_capacity),
+                ('data_timestamp', str(r.record_info.data_timestamp)),
+                ('data_offset', r.record_info.data_offset),
+                ('data_size', r.record_info.data_size),
+            ))
+
+        result[r.group_id] = group_result
+
+    print_result(result)
+
+
+def process_read():
+    """Process dnet_client read."""
+    parser = OptionParser()
+    parser.usage = '%prog write [options]'
+    parser.add_option('--key', dest='key', type='str',
+                      help='Key by which write should be executed')
+    parser.add_option('--hex', action='store_true', dest='hex', default=False,
+                      help='Specifies that the key used with --key is hex string')
+    parser.add_option('--data', action='store_true', dest='data', default=False,
+                      help='Force dnet_client to read data')
+    parser.add_option('--data-file', dest='data_file', metavar="FILE",
+                      help='Force dnet_client to read data and write it to @--data-file')
+    parser.add_option('--data-offset', dest='data_offset', default=0, type='int',
+                      help='Offset with which data should be read [default: %default]')
+    parser.add_option('--data-size', dest='data_size', default=0, type='int',
+                      help='Size of data part which should be read [default: %default]')
+    parser.add_option('--json', action='store_true', dest='json', default=False,
+                      help='Force dnet_client to read json')
+    parser.add_option('--json-file', dest='json_file', metavar='FILE',
```

Comment:
`metavar` is used only in help message and signify that the option accepts a filepath.